host-pool-service-blueprints/host-pool-service.yaml

```diff
@@ -0,0 +1,107 @@
+tosca_definitions_version: cloudify_dsl_1_1
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.2m8/types.yaml
+  - http://www.getcloudify.org/spec/fabric-plugin/1.2m8/plugin.yaml
+
+
+node_types:
+  cloudify.nodes.HostPoolService:
+    derived_from: cloudify.nodes.SoftwareComponent
+    properties:
+      directory:
+        type: string
+        default: /home/ubuntu/host_pool
+      environment:
+        # Accepted environment variables:
+        # * HOST_POOL_CONFIG,
+        # * NUM_WORKERS,
+        # * PID_FILE,
+        # * LOG_LEVEL,
+        # * LOG_FILE,
+        # * BIND_INTERFACE.
+        default: {}
+      ssh_host:
+        type: string
+      ssh_user:
+        type: string
+        default: ubuntu
+      ssh_key_path:
+        type: string
+        default: /home/ubuntu/.ssh/id_rsa
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          implementation: fabric.fabric_plugin.tasks.run_task
```

Comment:
the mapping should be to the script plugin, not the fabric plugin.
that is
create: scripts/tasks.py